src/core/commands.ts

```diff
@@ -609,16 +618,64 @@ export const updateDestination = async (payload: {
     (x) => destinationId === x.destinationId,
   )
   destination.address.rtmpPush = rtmpPush
```

Comment:
Not sure if it makes sense to include the previous value in the DestinationChanged event, but it could be helpful. Just a thought.

Reply:
I considered it but decided to follow pattern that Hudson started with `updateProjectMeta`